pgcontents/query.py

```diff
@@ -549,7 +550,8 @@ def save_file(db, user_id, path, content, encrypt_func, max_size_bytes):
     return res
 
 
-def generate_files(engine, crypto_factory, min_dt=None, max_dt=None):
+def generate_files(engine, crypto_factory, min_dt=None, max_dt=None,
+                   logger=None):
```

Comment:
Is there anywhere we actually pass a logger to this, or does that happen downstream?